client/packages/electron/forge.config.ts

```diff
@@ -35,6 +38,44 @@ const config: ForgeConfig = {
       },
     }),
   ],
+  hooks: {
```

Comment:
copy paste from here:

https://github.com/serialport/node-serialport/issues/2464#issuecomment-1444129576

Reply:
this is quite something! injecting yarn install into the electron build - quite the trick!

Reply:
I was in a rush so did it that way, potentially there is something else we can do. The problem btw is module not found in built version (but fine when running local dev)